generated_api_shadow/BUILD

```diff
@@ -145,7 +145,6 @@ proto_library(
         "//envoy/config/resource_monitor/injected_resource/v2alpha:pkg",
         "//envoy/config/retry/omit_canary_hosts/v2:pkg",
         "//envoy/config/retry/previous_hosts/v2:pkg",
-        "//envoy/config/retry/previous_hosts/v3:pkg",
```

Comment:
not sure why, but this seems wrong - and has come up on another PR

@htuch do you know why this keeps being added/removed ?

Reply:
I think if you merge master this will be gone. I'm not sure why it happened, i.e. how we merged something that still had this.